### ISSUE TITLE
Not jumping to line when pressing <Enter> on error line in location list

### ISSUE THREAD

**Issue by olalonde** [scrooloose/syntastic#1758]
![syntastic](https://cloud.githubusercontent.com/assets/416585/14938954/03653068-0eea-11e6-94cf-cd79b447e6de.gif)

I am not sure if this is an issue with Syntastic but recently my error list has been acting weird (when I select a line in the location list and click Enter, it doesn't jump to the error line). I can't recall when it first stopped working but maybe a few weeks ago (it was working fine before).

I have this problem with all my syntax checkers so it's not specific to eslint.

```
$ vim --version
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Mar 31 2016 00:38:28)
MacOS X (unix) version
Included patches: 1-1655
Compiled by Homebrew
```

**Comment by olalonde** [comment]
Fixed by upgrading syntastic... Dupe of https://github.com/scrooloose/syntastic/issues/1738 probably

**Comment by lcd047** [comment]
Not fixed, see the end of #1127 for details.
